pkg/platform/aws/aws.go

```diff
@@ -35,6 +35,7 @@ type workerPool struct {
 	Count        int               `hcl:"count"`
 	SSHPubKeys   []string          `hcl:"ssh_pubkeys"`
 	InstanceType string            `hcl:"instance_type,optional"`
+	OSArch       string            `hcl:"os_arch,optional"`
```

Comment:
This must also be documented.

Reply:
I've added the parameters to docs/configuration-reference/platforms/aws.md, is there another place I should document this?

Reply:
I think only `docs/configuration-reference/platforms/aws.md` is fine, thanks!